sass/utilities/variables.sass

```diff
@@ -27,8 +27,8 @@ $red:          hsl(348, 100%, 61%) !default
 
 // Typography
 $family-sans-serif: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", "Helvetica", "Arial", sans-serif !default
-$family-monospace: monospace !default
-$render-mode: optimizeLegibility !default
+$family-monospace: monospace !default
+$render-mode: optimizeLegibility !default
```

Comment:
as I said the only change in this file was only adding 2 variables that I mentioned and the IDE switched LF to CRLF so I reverted this IDE change in this commit! and as you can see at the top of the PR page the dirty lines are less than first commit now